lib/ioki/apis/operator_api.rb

```diff
@@ -370,6 +370,11 @@ class OperatorApi
         :cancellation_statement,
         base_path:   [API_BASE_PATH, 'products', :id],
         model_class: Ioki::Model::Operator::CancellationStatement
+      ),
+      Endpoints.crud_endpoints(
+        :ride_option,
+        base_path:   [API_BASE_PATH, 'products', :id],
+        model_class: Ioki::Model::Operator::RideOptions
```

Comment:
Not a change in this PR: should the model `RideOptions` rather be called `RideOption`?

Reply:
Yes! This should be `RideOption` 👍